releasedir/fs_blobs_dir.go

```diff
@@ -244,6 +244,8 @@ func (d FSBlobsDir) TrackBlob(path string, src io.ReadCloser) (Blob, error) {
 
 	blobs[idx] = Blob{Path: path, Size: fileInfo.Size(), SHA1: sha1}
 
+	tempFile.Close()
```

Comment:
Sure, but what should that test do? This method does [defer the close of the temp file][1], but now the defer close will fail if this line is reached. The defer close isn't checking for failure, so this close is causing a silent failure, but also preventing an error in Windows environments.

[1]: https://github.com/cloudfoundry/bosh-cli/pull/171/files/f78baee58902f252afde22804b19549c0618b805#diff-91f1299e44f3b411467c957c91b729bbL208